### ISSUE TITLE
Admin User is locked out -- can't find "users" directory to resolve

### ISSUE THREAD

**Issue by boredomdenied** [jellyfin/jellyfin#3980]
**Describe the bug**

I'm locked out of admin acct. Tried to test fail2ban jail for jellyfin. It didn't log the failed attempts, but I did get locked out anyways. Checking with fail2ban, my ip is not blocked. I can disable fail2ban and it makes no difference. It appears this is built in to jellyfin anyways. Found a supposed solution to set `isDisabled=false` and `invalidLoginAttempts=0` in `/etc/jellyfin/users`. I have searched all through the directory structure and can't find this. I have followed the main site for docker install:

`docker pull jellyfin/jellyfin:latest
mkdir -p /srv/jellyfin/{config,cache}
docker run -d -v /srv/jellyfin/config:/config -v /srv/jellyfin/cache:/cache -v /media:/media --net=host jellyfin/jellyfin:latest`

searching through `/srv/jellyfin` I can't find where the "users" would be. Please help me out here.

**System (please complete the following information):**
 - OS: Ubuntu 20.04LTS
 - Virtualization: Docker
 - Clients: Browser
 - Browser:Chrome 84
 - Jellyfin Version: latest
 - Reverse Proxy: nginx 

**To Reproduce**

- attempt multiple login fails
- locked out admin acct
- password reset
- entered pinfrom json file
- entered pin for login password

**Expected behavior**

I expect to be able to login. Instead I'm met with:

> You are not authorized to access the server at this time. Please contact your server administrator for more information.

**Logs**

In the database it states:
> User brandon has been locked outUserLockedOut

**Comment by whalehub** [comment]
@boredomdenied The user configs were recently moved from individual folders to the SQLite database file `jellyfin.db` in the `/data` folder. There's a table called `Users` in there which you can modify to get the desired result. On Windows, I can recommend the tool [DB Browser for SQLite](https://github.com/sqlitebrowser/sqlitebrowser) for this.

**Comment by boredomdenied** [comment]
Thank you. I was able to connect with @joshuaboniface on Jellyfin Troubleshooting via Matrix. He was able to give me the command to delete Users. I also toggled with boolean for `IsStartupWizardCompleted` to create a new Admin acct. Unfortunately the end result was a User that didn't have the Admin kind. I found another page which led me to mapping from the Id of the user to the Guid in the Permissions table. Shifting the Kind from '0' to '1' led to breaking the App. Perhaps I was too tired and messed something up. At any rate, I did wind up nuking the directories, restarting the docker container, and going through the StartupWizard another go. Only a few users were created so it's not a big deal for them to create new accounts.
